test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -94,55 +96,44 @@ var _ = BeforeSuite(func() {
 	}
 	masterSSHPrivateKeyFilepath = cfg.GetSSHKeyPath()
 	longRunningApacheDeploymentName = "php-apache-long-running"
+	kubeConfig, err = GetConfig()
+	Expect(err).NotTo(HaveOccurred())
+	sshConn, err = remote.NewConnection(kubeConfig.GetServerName(), masterSSHPort, eng.ExpandedDefinition.Properties.LinuxProfile.AdminUsername, masterSSHPrivateKeyFilepath)
```

Comment:
ditto for holding open a single ssh connection reference (to the first master in this case) for all test jobs